src/lib/core/api/DataClient/types.ts

```diff
@@ -557,16 +567,21 @@ export interface PieplotRequestParams {
   filters: Filter[];
   config: {
     outputEntityId: string;
-    showMissingness: 'TRUE' | 'FALSE';
+    showMissingness:
+      | 'TRUE'
+      | 'FALSE'
+      | 'noVariables'
+      | 'allVariables'
+      | 'strataVariables';
```

Comment:
Interested in the meaning of true/false vs noVariables/allVariables. Do they mean different things?

Reply:
True/false are essentially deprecated values now, but in the interest of not breaking loads of things.... 
The new values are the other three, where the old false has the same behavior as the new 'noVariables' and the old true has the same behavior as 'strataVariables'

Reply:
That makes sense! :)

Reply:
You got it from two sources, must be true!